pyproject.toml

```
@@ -24,41 +24,45 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Topic :: Utilities",
     "Topic :: Software Development :: Testing",
     "Operating System :: POSIX :: Linux",
     ]
-dependencies = [              # F39 / PyPI
-    "click>=8.0.3,!=8.1.4",   # 8.1.3 / 8.1.6 TODO type annotations tmt.cli.Context -> click.core.Context click/issues/2558
-    "docutils>=0.16",         # 0.16 is the current one available for RHEL9
+dependencies = [
+    # Minimal versions based on c9s rpms
+    "click>=8.0.3",
+    "docutils>=0.16",
     "fmf>=1.7.0",
-    "jinja2>=2.11.3",         # 3.1.2 / 3.1.2
-    "packaging>=20",          # 20 seems to be available with RHEL8
-    "pint>=0.16.1",           # 0.16.1
+    "jinja2>=2.11.3",
+    "packaging>=20.9",
+    "pint>=0.16.1",
     "pydantic>=1.10.14",
-    "pygments>=2.7.4",        # 2.7.4 is the current one available for RHEL9
-    "requests>=2.25.1",       # 2.28.2 / 2.31.0
-    "ruamel.yaml>=0.16.6",    # 0.17.32 / 0.17.32
-    "urllib3>=1.26.5, <3.0",  # 1.26.16 / 2.0.4
-    "typing-extensions>=4.9.0; python_version < '3.13'",
+    "pygments>=2.7.4",
+    "requests>=2.25.1",
+    "ruamel.yaml>=0.16.6",
+    "urllib3>=1.26.5, <3.0",
+    "typing-extensions>=4.12.2; python_version < '3.13'",
     ]
 
 [project.optional-dependencies]
 ansible = [
-    "ansible-core",
+    "ansible-core>=2.14.18",
     ]
 test-convert = [
     "html2text>=2020.1.16",
     "markdown>=3.3.4",
-    "nitrate>=1.8.2",
+    "nitrate>=1.9.0",
     "python-bugzilla>=3.2.0",
     ]
 export-polarion = [
-    "pylero>=0.0.8",
+    "pylero>=0.1.0",
     ]
 provision-beaker = [
-    "mrack>=1.23.1",
+    "mrack>=1.23.3",
```

```suggestion
    "mrack>=1.23.2",
```

This is not going to work, according to PyPI, the most recent version is     `1.23.2`.